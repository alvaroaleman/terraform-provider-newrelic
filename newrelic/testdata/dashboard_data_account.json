{
  "description": "The Dashboard",
  "name": "Dashboard",
  "pages": [
    {
      "description": "Overview",
      "name": "Overview",
      "widgets": [
        {
          "layout": {
            "column": 1,
            "height": 3,
            "row": 1,
            "width": 6
          },
          "linkedEntityGuids": null,
          "rawConfiguration": {
            "nrqlQueries": [
              {
                "accountIds": [
                  123
                ],
                "query": "SELECT foo FROM Metric TIMESERIES"
              },
              {
                "accountIds": [
                  123
                ],
                "query": "SELECT bar FROM Metric TIMESERIES"
              }
            ]
          },
          "title": "title",
          "visualization": {
            "id": "viz.table"
          }
        },
        {
          "layout": {
            "column": 7,
            "height": 3,
            "row": 1,
            "width": 6
          },
          "rawConfiguration": {
            "nrqlQueries": [
              {
                "accountIds": [
                  123
                ],
                "query": "SELECT foo FROM Metric TIMESERIES"
              },
              {
                "accountIds": [
                  123
                ],
                "query": "SELECT bar FROM Metric TIMESERIES"
              }
            ]
          },
          "title": "Other Title",
          "visualization": {
            "id": "viz.table"
          }
        }
      ]
    }
  ],
  "permissions": "PUBLIC_READ_WRITE",
  "variables": [
    {
      "defaultValues": [
        {
          "value": {
            "string": "*"
          }
        }
      ],
      "isMultiSelection": true,
      "items": null,
      "name": "env",
      "nrqlQuery": {
        "accountIds": [
          123
        ],
        "query": "SELECT uniques(`deployment.environment`) FROM Metric"
      },
      "replacementStrategy": "DEFAULT",
      "title": null,
      "type": "NRQL"
    },
    {
      "defaultValues": [
        {
          "value": {
            "string": "*"
          }
        }
      ],
      "isMultiSelection": true,
      "items": null,
      "name": "sync_env",
      "nrqlQuery": {
        "accountIds": [
          123
        ],
        "query": "SELECT uniques(`sync_env`) FROM Metric"
      },
      "replacementStrategy": "DEFAULT",
      "title": null,
      "type": "NRQL"
    }
  ]
}
