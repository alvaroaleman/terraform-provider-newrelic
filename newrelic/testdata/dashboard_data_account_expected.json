{"description":"The Dashboard","name":"Dashboard","pages":[{"description":"Overview","name":"Overview","widgets":[{"configuration":{},"layout":{"column":1,"height":3,"row":1,"width":6},"linkedEntityGuids":null,"rawConfiguration":{"nrqlQueries":[{"accountIds":[456,789],"query":"SELECT foo FROM Metric TIMESERIES"},{"accountIds":[456,789],"query":"SELECT bar FROM Metric TIMESERIES"}]},"title":"title","visualization":{"id":"viz.table"}},{"configuration":{},"layout":{"column":7,"height":3,"row":1,"width":6},"linkedEntityGuids":null,"rawConfiguration":{"nrqlQueries":[{"accountIds":[456,789],"query":"SELECT foo FROM Metric TIMESERIES"},{"accountIds":[456,789],"query":"SELECT bar FROM Metric TIMESERIES"}]},"title":"Other Title","visualization":{"id":"viz.table"}}]}],"permissions":"PUBLIC_READ_WRITE","variables":[{"defaultValues":[{"value":{"string":"*"}}],"isMultiSelection":true,"nrqlQuery":{"accountIds":[456,789],"query":"SELECT uniques(`deployment.environment`) FROM Metric"},"name":"env","replacementStrategy":"DEFAULT","type":"NRQL"},{"defaultValues":[{"value":{"string":"*"}}],"isMultiSelection":true,"nrqlQuery":{"accountIds":[456,789],"query":"SELECT uniques(`sync_env`) FROM Metric"},"name":"sync_env","replacementStrategy":"DEFAULT","type":"NRQL"}]}